### PR TITLE
Feasibility of Group Header is implemented -  Grid

### PR DESCRIPTION
**Pull request by ggkrishkumar**

### CODE REVIEW

File: grid/src/Overlays/exportdata/index.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ const ExportData = memo((props) => {
         additionalColumn
     } = props;
 
+    const updatedOriginalColumns = [];
+    originalColumns.forEach((item) => {
+        item.columns.forEach((col) => {
+            updatedOriginalColumns.push(col);
+        });
+    });
+    console.log(updatedOriginalColumns);
```
Comment (iCargopoc) [inline grid/src/Overlays/exportdata/index.js]: Remove unwanted console logs

File: grid/src/Utilities/ColumnsUtilities.js [hunk 2]
```diff
@@ -30,6 +31,7 @@ export const extractColumns = (
         // If there are no copies of original Cells create a new copy from Inner cells
         if (!isOriginalInnerCellsPresent && isInnerCellsPresent) {
             elem.originalInnerCells = innerCells;
+            // elem.column = "test";
```
Comment (iCargopoc) [inline grid/src/Utilities/ColumnsUtilities.js]: Remove unwanted code

File: grid/src/Overlays/exportdata/index.js [hunk 3]
```diff
@@ -23,6 +23,14 @@ const ExportData = memo((props) => {
         additionalColumn
     } = props;
 
+    const updatedOriginalColumns = [];
```
Comment (iCargopoc) [inline grid/src/Overlays/exportdata/index.js]: Move logic to create updatedOriginalColumns and re-use it in for all 3 overlays

**Review by iCargopoc — CHANGES_REQUESTED** [review]

File: grid/src/Overlays/exportdata/index.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ const ExportData = memo((props) => {
     );
 
     // List of columns + expand based on user selection from manage overlay
-    const updatedColumnsPerUserSelection = [...columns].concat(
+    const updatedColumnsPerUserSelection = [...originalColumns].concat(
```
Comment (iCargopoc) [inline grid/src/Overlays/exportdata/index.js:42]: Should this be really changed?

**Review by iCargopoc — CHANGES_REQUESTED** [review]

File: grid/src/Overlays/exportdata/index.js (hunk 4 above)
Comment (ggkrishkumar) [reply grid/src/Overlays/exportdata/index.js:42]: Yes.

**Review by iCargopoc — APPROVED** [review]

**Pull request merged**